azure-cosmos/src/test/groovy/io/micronaut/azure/cosmos/client/CosmosClientSpec.groovy

```diff
@@ -0,0 +1,58 @@
+package io.micronaut.azure.cosmos.client
+
+import com.azure.cosmos.CosmosAsyncClient
+import com.azure.cosmos.CosmosClient
+import com.azure.cosmos.CosmosContainer
+import com.azure.cosmos.CosmosDatabase
+import com.azure.cosmos.models.CosmosContainerProperties
+import com.azure.cosmos.models.CosmosContainerResponse
+import com.azure.cosmos.models.CosmosDatabaseResponse
+import com.azure.cosmos.models.CosmosItemRequestOptions
+import com.azure.cosmos.models.CosmosItemResponse
+import com.azure.cosmos.models.PartitionKey
+import io.micronaut.context.ApplicationContext
+import spock.lang.AutoCleanup
+import spock.lang.IgnoreIf
+import spock.lang.Shared
+import spock.lang.Specification
+
+@IgnoreIf({ env["GITHUB_WORKFLOW"] })
+class CosmosClientSpec extends Specification implements AzureCosmosTestProperties {
+
+    @AutoCleanup
+    @Shared
+    ApplicationContext context = ApplicationContext.run(properties)
+
+    void "should get cosmos client and perform write/read operations"() {
+        when:
+            CosmosClient client = context.getBean(CosmosClient)
```

Comment:
This indenting is evil! 😜

Please align all code at the same level as the `when/then` labels.

Reply:
I have seen different indenting in different tests and liked this better, but will update it.

Reply:
I have changed it per your suggestion Alvaro, and I will assume from now on all tests should be indented that way.

Reply:
Thanks! And point me to anyone who thinks otherwise!

Reply:
😶 You need to convince IntelliJ to change the default setting